doc/source/api.rst

```diff
@@ -233,9 +238,14 @@ The VirtualCircuit object
 
         a unique identifer for this instance: :data:`((host, port), priority)`
 
+    .. attribute:: command_queue
+
+        The queue from which higher levels feed Commands from
```

Comment:
This phrase tripped me up reading it. How about, "A queue where incoming Commands are put, to get read out by higher layers"